### DISCUSSION TITLE
注册成Windows服务 遇到报错

### DISCUSSION THREAD

**Discussion by KaiserMatt721** [1715173329/UnblockNeteaseMusic#1070] [Q&A] [answered]
已经安装了node-windows但还是报错，求大佬解惑
报错信息：
C:\WINDOWS\system32> node "D:\Program Files (x86)\netease unblock\server-0.27.1\nw.js"
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'node-windows'
Require stack:
- D:\Program Files (x86)\netease unblock\server-0.27.1\nw.js
[90m    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)[39m
[90m    at Module._load (node:internal/modules/cjs/loader:920:27)[39m
[90m    at Module.require (node:internal/modules/cjs/loader:1141:19)[39m
[90m    at require (node:internal/modules/cjs/helpers:110:18)[39m
    at Object.<anonymous> (D:\Program Files (x86)\netease unblock\server-0.27.1\nw.js:1:17)
[90m    at Module._compile (node:internal/modules/cjs/loader:1254:14)[39m
[90m    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)[39m
[90m    at Module.load (node:internal/modules/cjs/loader:1117:32)[39m
[90m    at Module._load (node:internal/modules/cjs/loader:958:12)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)[39m {
  code: [32m'MODULE_NOT_FOUND'[39m,
  requireStack: [ [32m'D:\\Program Files (x86)\\netease unblock\\server-0.27.1\\nw.js'[39m ]
}

Node.js v18.16.0

**Comment by 1715173329** [comment, marked as answer]
fyi https://github.com/UnblockNeteaseMusic/server/issues/1039#issuecomment-1492429556

**Reply by KaiserMatt721** [reply]
感谢指路，不过我在过来求助之前已经看到这个并且尝试过了，还是不行，上面那些就是在执行了这个之后显示的😢

**Reply by KaiserMatt721** [reply]
再次感谢，我在又看了一遍之后发现用Power shell窗口执行成功了，之前用直接打开的cmd一直不行，不太理解是为什么
